src/modules/core/components/FriendlyName/FriendlyName.tsx

```diff
@@ -33,6 +33,10 @@ const FriendlyUserName = ({
 }: Props) => {
   const addressRef = useRef<HTMLElement>(null);
 
+  const { data: colonyData } = useColonyQuery({
+    variables: { address: walletAddress || '' },
+  });
```

Comment:
Please don't fetch data in this component, it's pointless. Just pass it along the same way we do the user profile.

That way you can still keep your display logic intact.